.eslintrc.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+    webextensions: true
+  },
+  extends: [
+    "eslint:recommended",
+    "plugin:mozilla/recommended"
+  ],
+  parserOptions: {
+    ecmaVersion: 2017
+  },
+  plugins: [
+    "mozilla"
+  ],
+  root: true,
+  rules: {
+    "eqeqeq": "error",
+    "no-var": "error",
+    "prefer-const": "error",
+    "quotes": ["error", "double"],
```

Comment:
These are just some basic rules. We can add as many as you want (ie: flag "TODO" comments as warnings/errors, require JSDoc comments, etc), but this is probably a bare minimum of what we'd want.